README.md

```diff
@@ -11,14 +11,13 @@
 
 ### Discover BitTorrent and WebTorrent peers
 
-This module bundles [bittorrent-dht](https://www.npmjs.com/package/bittorrent-dht) and
-[bittorrent-tracker](https://www.npmjs.com/package/bittorrent-tracker) clients and exposes a
-single API for discovering BitTorrent peers via both discovery methods.
+This module bundles [bittorrent-dht](https://www.npmjs.com/package/bittorrent-dht),
+[bittorrent-tracker](https://www.npmjs.com/package/bittorrent-tracker) and [bittorrent-lsd](https://www.npmjs.com/package/bittorrent-lsd) clients and exposes a single API for discovering BitTorrent peers via both discovery methods.
 
 ## features
 
 - simple API
-- find peers from trackers and the DHT
+- find peers from trackers, DHT and LSD
```

Comment:
super nit-picky, but I like to put commas after "and" like this:

"find peers from trackers, DHT, and LSD"

Reply:
Changed! 👍